kubelet/datadog_checks/kubelet/summary.py

```diff
@@ -145,10 +149,14 @@ def _report_system_container_metrics(self, stats, instance_tags):
                 cpu_usage = ctr.get('cpu', {}).get('usageNanoCores')
                 if cpu_usage:
                     self.gauge(self.NAMESPACE + '.runtime.cpu.usage', cpu_usage, instance_tags)
+                memory_usage = ctr.get('memory', {}).get('usageBytes')
```

Comment:
It appears this line is never used

Reply:
Thanks @vboulineau for spotting that, modified accordingly!